aplib.net-demo/Assets/Scripts/Inventory/Key.cs

```diff
@@ -0,0 +1,18 @@
+public class Key : Item
+{
+    /// <summary>
+    /// The key ID, to check which door it can open
+    /// </summary>
+    public int id;
+    /// <summary>
+    /// the constructor that gives the key its Id
```

Comment:
Behavior vs implementation.
```suggestion
    /// Initializes a key with a given ID.
```